examples/promote_observable_to_indicator.py

```diff
@@ -0,0 +1,25 @@
+# coding: utf-8
+
+from pycti import OpenCTIApiClient
+
+# Variables
+api_url = "http://opencti:4000"
+api_token = "bfa014e0-e02e-4aa6-a42b-603b19dcf159"
+
+# OpenCTI initialization
+opencti_api_client = OpenCTIApiClient(api_url, api_token)
+
+# DEPRECATED >=6.2 & <6.5. Use `promote_to_indicator_v2` instead
+obs1 = opencti_api_client.stix_cyber_observable.create(
```

Comment:
this example file could be updated ; we probably want to showcase only the latest version. But this is handy for testing the PR.